visualization/app/codeCharta/ui/maxTreeMapFiles/maxTreeMapFiles.component.spec.ts

```diff
@@ -0,0 +1,40 @@
+import "./maxTreeMapFiles.module"
+import { MaxTreeMapFilesController } from "./maxTreeMapFiles.component"
+import { instantiateModule, getService } from "../../../../mocks/ng.mockhelper"
+import { IRootScopeService } from "angular"
+import { StoreService } from "../../state/store.service"
+import { MaxTreeMapFilesService } from "../../state/store/appSettings/maxTreeMapFiles/maxTreeMapFiles.service"
+import { maxTreeMapFiles } from "../../state/store/appSettings/maxTreeMapFiles/maxTreeMapFiles.reducer"
+import _ from "lodash"
+
+describe("MaxTreeMapFilesController", () => {
```

Comment:
Add tests for the methods `onMaxTreeMapFilesChanged()` and `onChangeMaxTreeMapFilesSlider()`. You can have a look at the `layoutSelection.component.spec.ts`, which is quite similar